package.json

```
@@ -124,9 +125,9 @@
     "react-moment-proptypes": "^1.6.0",
     "react-outside-click-handler": "^1.2.4",
     "react-portal": "^4.2.0",
-    "react-with-direction": "^1.3.0",
-    "react-with-styles": "^3.2.3",
-    "react-with-styles-interface-css": "^4.0.3"
+    "react-with-direction": "^1.3.1",
```

not a huge deal, but this should probably match the peer dep range to avoid dupes
```suggestion
    "react-with-direction": "^1.3.0",
```

---

I've updated the peer dep as well. It's a breaking change.